stakerdb/trackedtranactionstore.go

```diff
@@ -552,6 +552,104 @@ func (c *TrackedTransactionStore) addTransactionInternal(
 	})
 }
 
+func CreateTrackedTransaction(
+	btcTx *wire.MsgTx,
+	stakingOutputIndex uint32,
+	stakingTime uint16,
+	fpPubKeys []*btcec.PublicKey,
+	pop *ProofOfPossession,
+	stakerAddress btcutil.Address,
+) (*StoredTransaction, error) {
+	serializedTx, err := utils.SerializeBtcTransaction(btcTx)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(fpPubKeys) == 0 {
+		return nil, fmt.Errorf("cannot add transaction without finality providers public keys")
+	}
+
+	var fpPubKeysBytes [][]byte = make([][]byte, len(fpPubKeys))
```

Comment:
```suggestion
	fpPubKeysBytes := make([][]byte, len(fpPubKeys))
```
No need to be redundant with type declaration